finality-aleph/src/sync/message_limiter.rs

```diff
@@ -0,0 +1,181 @@
+use parity_scale_codec::Encode;
+
+use crate::sync::data::MAX_SYNC_MESSAGE_SIZE;
+
+pub const MSG_BYTES_LIMIT: usize = MAX_SYNC_MESSAGE_SIZE as usize;
```

Comment:
do we need two separate consts?

Reply:
I think we need to set the limits properly this time... Please check if e.g. this is accurate: https://substrate.stackexchange.com/questions/6999/what-is-the-maximum-size-of-a-block

If possible, let's import relevant consts from Substrate, and use them only to introduce some static assertions (we define the limits, and the compiler tells us if they decide to change sth - this is imho the best approach).

Reply:
We could use only one constant :)

Reply:
I left this here because it serve as convenient place for cast to usize

Reply:
terrible :D